includes/class-amp-theme-support.php

```diff
@@ -754,17 +754,27 @@ public static function add_hooks() {
 		} );
 
 		/*
-		 * Add additional markup required by AMP <https://www.ampproject.org/docs/reference/spec#required-markup>.
-		 * Note that the meta[name=viewport] is not added here because a theme may want to define one with additional
-		 * properties than included in the default configuration. If a theme doesn't include one, then the meta viewport
-		 * will be added when output buffering is finished. Note that meta charset _is_ output here because the output
-		 * buffer will need it to parse the document properly, and it must be exactly as is to be valid AMP. Nevertheless,
-		 * in this case too we should defer to the theme as well to output the meta charset because it is possible the
-		 * install is not on utf-8 and we may need to do a encoding conversion.
+		 * "AMP HTML documents MUST contain the AMP boilerplate code (head > style[amp-boilerplate] and noscript > style[amp-boilerplate]) in their head tag."
+		 * https://www.ampproject.org/docs/fundamentals/spec#required-markup
```

Comment:
@westonruter I'm wondering if we want to standardize our inline link references to this:

```
"AMP HTML documents MUST contain the AMP boilerplate code (head > style[amp-boilerplate] and noscript > style[amp-boilerplate]) in their head tag." {@link https://www.ampproject.org/docs/fundamentals/spec#required-markup AMP Required Markup}.

What do you think?

Reply:
@westonruter True.  I'm more thinking about consistency of _how we document_ links whether in a DocBlock, file header, or inline comment.  We have a mixture of documenting techniques:

- `{@see http Document Title}`
- `See http....`
- `see <http....>`
- etc.

This is a larger discussion on our coding standards to ensure we have consistency.